docs_guide/getting_started.rst

```diff
@@ -1,3 +1,5 @@
+.. _getting_started:
```

Comment:
```suggestion
.. _getting-started:
```
I think the convention is to use `-` instead of `_` in the labels.

Reply:
Not sure if there's a properly stablished convention. While it's true that [Sphinx's documentation](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-ref) uses `-`, [Read the Docs](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-ref) uses whitespaces ` ` and [this tutorial](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-ref) uses `_`, for example.

When I wrote that I stuck to the names of the files, that used `_`, but I have no problem with using `-` instead.

Reply:
@Guillermo-Mijares-Vilarino All the links you put above are the same one. Can you send the correct links here? I think it also makes sense to use the file name. Let me think about it.

Reply:
Fixed the Read the Docs link

Reply:
Tutorial fixed now too